pkg/cli/newapp/newapp.go

```diff
@@ -369,7 +369,13 @@ func (o *AppOptions) RunNewApp() error {
 	if len(result.Name) > 0 {
 		// only set the computed implicit "app" label on objects if no object we've produced
 		// already has the "app" label.
-		appLabel := map[string]string{"app": result.Name}
+		appLabel := map[string]string{
+			"app":                         result.Name,
+			"app.kubernetes.io/name":      result.Name,
```

Comment:
This is typically the name of the builder image.

Reply:
Done here 
9734a87